### PR TITLE
adding a --windows option for hab plan init to generate powershell plan

### LINKED ISSUE #4527: Implement `hab plan init` for Powershell plans
This was never done and as I started to write the Windows demo, it bacame obviously necessary to have it.

### PR DESCRIPTION
**Pull request by mwrock**
Will need this for the powershell based tutorial. Fixes #4527

Signed-off-by: mwrock <matt@mattwrock.com>

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

File: www/source/partials/docs/_dev-pkgs-write-plans.html.md.erb [hunk 1]
```diff
@@ -61,6 +61,14 @@ All plans must have a `plan.sh` at the root of the plan context. This file will
 
     will result in a new `habitat` directory located at `/path/to/<reponame>/habitat`. A plan.sh file will be created and the `pkg_name` variable in plan.sh will be set to _\<reponame\>_. Also, any environment variables that you have previouly set (such as `HAB_ORIGIN`) will be used to populate the respective `pkg_*` variables.
 
+    If you want to generate a Powershell based plan file for building Windows packages, use the `--windows` option:
+
+    ```Powershell
+    PS C:\\> hab plan init --windows yourplan
+    ```
+
+    this generates a `plan.ps1` with propper Powershell variables, arrays, hashtables and functions.
```
Comment (cnunciato) [inline www/source/partials/docs/_dev-pkgs-write-plans.html.md.erb]: Little typo: `proper`

File: components/hab/static/full_template_plan.ps1 [hunk 2]
```diff
@@ -0,0 +1,419 @@
+{{#if with_docs ~}}
+# This file is the heart of your application's habitat.
+# See full docs at https://www.habitat.sh/docs/reference/plan-syntax/
+
+# Required.
+# Sets the name of the package. This will be used in along with `$pkg_origin`,
```
Comment (cnunciato) [inline components/hab/static/full_template_plan.ps1]: Should this be `used along with`?

**Comment by mwrock** [comment]
@thesentinels approve

**Comment by thesentinels** [comment]
:metal: I am testing your branch against master before merging it. We do this to ensure that the master branch is never failing tests.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started testing this PR](https://travis-ci.org/habitat-sh/habitat/builds/335885353).

**Comment by thesentinels** [comment]
:sparkling_heart: Travis CI [reports this PR passed](https://travis-ci.org/habitat-sh/habitat/builds/335885353).

It always makes me feel nice when humans approve of one anothers work. I'm merging this PR now.

I just want you and the contributor to answer me one question:

gif-keyboard-3280869874741411265

**Pull request merged**